app/views/thesis/new.html.erb

```diff
@@ -3,55 +3,74 @@
 
-    <%= f.association :degrees, label_method: :name, as: :check_boxes,
+    <%= f.association :degrees, label_method: :name, as: :select,
                          include_hidden: false,
-                         wrapper_html: { class: 'field-wrap list-checkboxes' },
-                         input_html: { class: 'field-checkbox' } %>
-
-    <%= f.association :right, label_method: :statement, as: :radio_buttons,
-                              wrapper_html: { class: 'field-wrap list-radios' },
-                              input_html: { class: 'field-checkbox' } %>
+                         wrapper_html: { class: 'field-wrap select' },
+                         input_html: { class: 'field-select', multiple: false,
+                           data: { msg: 'Selecting your primary Degree is required.'}
+                         },
+                         hint: 'Select your primary degree as listed on your title page' %>
+
+    <%= f.association :right, label: 'Copyright',
+                              label_method: :statement, as: :select,
+                              include_hidden: false,
+                              wrapper_html: { class: 'field-wrap select' },
+                              input_html: { class: 'field-select',
+                                            multiple: false,
+                                            data: { msg: 'Selecting the appropriate copyright is required.'} },
+                              hint: 'For more information about thesis copyright please review the Copyright section of the <a href="https://libraries.mit.edu/archives/thesis-specs/#copyright">Specifications for Thesis Preparation</a>.'.html_safe %>
 
     <fieldset>
-      <legend>Graduation Date</legend>
+      <legend>Degree Date</legend>
 
         <div class='group-inline'>
 
-        <%= f.input :graduation_month, label: 'Month', as: :select,
-                             collection: Thesis::VALID_MONTHS,
-                             wrapper_html: { class: 'field-wrap' },
-                             input_html: { class: 'field field-select' } %>
+          <%= f.input :graduation_month, label: 'Month', as: :select,
+                               collection: Thesis::VALID_MONTHS,
+                               wrapper_html: { class: 'field-wrap' },
+                               input_html: { class: 'field field-select', data: { msg: 'Selecting Month of graduation is required.' } } %>
 
-        <%= f.input :graduation_year, label: 'Year', as: :string,
-                            wrapper_html: { class: 'field-wrap' },
-                            input_html: { class: 'field field-select' } %>
+          <%= f.input :graduation_year, label: 'Year', as: :string,
+                              wrapper_html: { class: 'field-wrap' },
+                              input_html: { class: 'field field-select', data: { msg: 'Selecting Year of graduation is required.' } } %>
```

Comment:
"Entering" instead of "Selecting" - and can we have it validate that it's a number or is that more trouble than it's worth?

Reply:
It does on the backend but we should up front too. Will fix.